packages/compute-baseline/src/baseline/support.ts

```diff
@@ -13,6 +13,18 @@ export function support(feature: Feature, browsers: Browser[]): Support {
   const support: Support = new Map();
   for (const b of browsers) {
     const releases = feature.supportedBy({ only: [b] });
+    // TODO:
+    // let lastInitial: Release | undefined;
+    // let lastInitialBoundary: "" | "≤" | undefined;
+
+    // const reverseChronological = b.releases.slice().reverse();
+    // let previousRelease: string | undefined;
+    // for (let index = 0; index < reverseChronological.length; index++) {
```

Comment:
Since we're iterating an index anyway, I think we can just iterate `b.releases` backwards like this:

```suggestion
    // for (let index = b.releases.length - 1; index >= 0; index--) {
```